src/system-plugins/software-update-alert/public/js/software-update.js

```diff
@@ -68,7 +68,7 @@
     var logoTitleModel = {};
     logoTitleModel.cockpitVersion = ko.observable('N/A');
     logoTitleModel.bbSerial = ko.observable('N/A');
-    $('a.brand').attr('data-bind', 'attr: {title: "Version: " + cockpitVersion() + "\\nBB Serial: " + bbSerial()}');
+    $('html /deep/ a.brand').attr('data-bind', 'attr: {title: "Version: " + cockpitVersion() + "\\nBB Serial: " + bbSerial()}');
```

Comment:
Having /deep/ references in the code implies someone had to go dig in to the shadow dom and is messing with DOM elements that are not intentionally exposed and thus may break in future updates.  Are these all stop gaps until we finish baking in the api/attributes for proper extension?